src/components/PDF/ApplicationForm.tsx

```diff
@@ -267,17 +297,27 @@ class ApplicationForm extends Component<Props, State> {
               <div className="progress-bar" role="progressbar" aria-valuenow={fillAmt} aria-valuemin={0} aria-valuemax={100} style={{ width: `${fillAmt}%` }}>{`${fillAmt}%`}</div>
             </div>
             <div className="container col-lg-10 col-md-10 col-sm-12">
-              <h2>Application Form Name</h2>
-              <p>Sample description of what this form is about. Or instructions for the form can go here.</p>
+              <h2>{ title }</h2>
+              <p>{ description }</p>
             </div>
           </div>
           <div className="container border px-5 col-lg-10 col-md-10 col-sm-12">
             <form onSubmit={this.onSubmitFormQuestions}>
               {formQuestions.map(
                 (entry) => (
-                  <div className="my-5" key={uuid()}>
+                  <div className="my-5" key={entry.fieldID}>
                     {
                       (() => {
+                        if (entry.fieldType === 'ReadOnlyField') {
+                          // TODO: Make it work
```

Comment:
Why is this all commented out? Can you please remove commented out code that's not needed, otherwise, it makes people confused.

Reply:
It was there for the future, but I fixed it already

Reply:
Could we do this in the future, including leaving comments with code for us to work on in the future, and if not then what should we do instead?